hydrus/app.py

```diff
@@ -38,7 +38,7 @@
 except Exception:
     pass
 
-AUTH = True
+AUTH = False
```

Comment:
Why are we changing it?

Reply:
Oh, Sorry I made it for testing purpose, slipped out of my mind to revert it back.